content/concepts/tokens/nfts/index.md

```diff
@@ -0,0 +1,46 @@
+---
+html: nft-index.html
```

Comment:
It would break fewer links and make migration easier if this file kept the `non-fungible-tokens.html` output filename.